packages/symfony-static-dumper/src/Command/DumpStaticSiteCommand.php

```diff
@@ -5,39 +5,56 @@
 namespace Symplify\SymfonyStaticDumper\Command;
 
 use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
 use Symfony\Component\Console\Output\OutputInterface;
 use Symfony\Component\DependencyInjection\ParameterBag\ParameterBagInterface;
 use Symplify\PackageBuilder\Console\Command\AbstractSymplifyCommand;
 use Symplify\SymfonyStaticDumper\Application\SymfonyStaticDumperApplication;
+use Symplify\SymfonyStaticDumper\Enum\Option;
 
 final class DumpStaticSiteCommand extends AbstractSymplifyCommand
 {
-    private string $publicDirectory;
-
-    private string $outputDirectory;
+    private string $projectDir;
 
     public function __construct(
         private SymfonyStaticDumperApplication $symfonyStaticDumperApplication,
         ParameterBagInterface $parameterBag
     ) {
-        parent::__construct();
-
-        $projectDir = (string) $parameterBag->get('kernel.project_dir');
+        $this->projectDir = (string) $parameterBag->get('kernel.project_dir');
 
-        $this->publicDirectory = $projectDir . '/public';
-        $this->outputDirectory = getcwd() . '/output';
+        parent::__construct();
     }
 
     protected function configure(): void
     {
         $this->setName('dump-static-site');
         $this->setDescription('Dump website to static HTML and CSS in the output directory');
+
+        // Adding arguments options for the main command
+        $this->addOption(
+            Option::PUBLIC_DIRECTORY,
+            null,
+            InputOption::VALUE_REQUIRED,
+            'Define the input public directory',
+            $this->projectDir . '/public'
```

Comment:
The original default value is here:

```php
    $this->publicDirectory = $projectDir . '/public';
```

Reply:
On the second thought, this could be also `getcwd()` directory. That would make more sense and make it simpler :+1: